onnxruntime/test/python/quantization/op_test_utils.py

```diff
@@ -68,6 +68,21 @@ def check_op_type_count(testcase, model_path, **kwargs):
         )
 
 
+def check_conv_per_channel_counts(testcase, model_path, node_name: str = "conv_node_quant"):
+    model = onnx.load(Path(model_path))
+    axis = 0
+    # minimum number of element in dims is -1
+    count = -2
```

Comment:
as the model only has Conv node, i think you can just pass the Channel Count in.